newIDE/app/src/MainFrame/EditorContainers/HomePage/InAppTutorials/GuidedLessons.js

```diff
@@ -217,7 +193,6 @@ const GuidedLessons = ({ selectInAppTutorial, lessonsIds }: Props) => {
                     title={item.title}
                     description={item.description}
                     durationInMinutes={item.durationInMinutes}
-                    keyPoints={item.keyPoints}
```

Comment:
You keep the props in the component `InAppTutorialPhaseCard`?

Reply:
Yes because, it's valuable for the Fling Game I think, there are more concepts to grasp